components/Map/Map.styled.tsx

```diff
@@ -0,0 +1,24 @@
+import { styled } from "@/stitches";
+import {
+  indigo,
+  indigoDarkA,
+  slate,
+  slateA,
+  slateDarkA,
+} from "@radix-ui/colors";
+import {ContainerStyled} from "../Shared/Container";
+
+const MapStyled = styled("div", {
+  height: "500px",
+  ".map-container": {
+    height: "500px",
+    width: "100%",
```

Comment:
There is a zIndex issue where the map rolls over top of the header. This should resolve it and make the map more fluid to the size of the screen.

```suggestion
  height: "100vh",
  position: "relative",
  zIndex: "0",

  ".map-container": {
    height: "100%",
    width: "100%",
```